2363/3/sinatra.txt

```diff
@@ -0,0 +1,5 @@
+gem 'shotgun'
+gem 'thin'
+--servers=thin
+padrino sintra models 
+gem sitekick мультипоточность
```

Comment:
Huh, why do you need this file? Isn't it better move this to README?

Reply:
Ops, that txt from lecture about sinatra